cdc/sink/mq.go

```diff
@@ -42,7 +42,11 @@ type mqEvent struct {
 	resolvedTs uint64
 }
 
-const defaultPartitionInputChSize = 12800
+const (
+	defaultPartitionInputChSize = 12800
+	// -1 means broadcast to all partition, it's the default for the default open protocol.
+	defaultDDLDispatchPartition = -1
```

Comment:
We also use this value in EmitCheckpointTs, can you change it together?

Reply:
this is only for ddl, I think we should have a further discussion about how to dispatch checkpointTs.

Reply:
I understand that in the current implementation, we also broadcast the checkpoint. I mean let it represent the broadcast, it doesn't change the current implementation. It just treats it as a constant. For example it could be named: defaultBrodcastPartitionNum.